services/resource-service/server/src/main/resources/distribution/conf/resources.json

```diff
@@ -59,5 +59,10 @@
     "type": "DROPBOX",
     "resourceId": "",
     "resourcePath": ""
+  },
+  {
+    "type": "GDRIVE",
+    "resourceId": "gdrive-file",
+    "resourcePath": "nsagdrive.txt"
```

Comment:
Google drive allows same directory/file name in the same path. Is there need of the path here?

Reply:
So we have kept same name for the key "resourcePath" consistent with other resources . It expects file name only. Path of the file is not needed.

Reply:
Use the GDrive specific names for the resource parameters. It's not required to have the resourcePath for all resource types. Only requirements is to have a resourceId field